packages/core/src/index.ts

```diff
@@ -22,5 +22,6 @@ export * from './client'
 export * from './operator'
 export * from './query'
 export * from './server'
+export * from './__tests__/minmodel'
```

Comment:
Why? :)

Reply:
Alternative is to have a copy of minmodel inside query package.